.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
+}
```

Comment:
`@babel/plugin-proposal-object-rest-spread`
`@babel/plugin-transform-arrow-functions`

the `@babel/preset-env` should target node

Reply:
I have added the plugins and made sure @babel/preset-env targets node